translations/schema2josmpreset.js

```diff
@@ -75,44 +75,38 @@ Object.keys(objs).forEach(s => {
     var presets = builder.create('presets');
 
     objs[s].schema.forEach(i => {
-        //Must de-dup the items and build up the geom array
-        if (items[i.desc]) {
-            items[i.desc].geoms.push(i.geom);
-        } else {
-
-            items[i.desc] = {
-                fcode: i.fcode,
-                geoms: [i.geom],
-                columns: i.columns,
-                hashes: []
-            };
-
-            //Build de-duped map of unique enumerations (list entry),
-            //combo and text elements keyed by hash
-            i.columns.forEach(col => {
-                let colHash = 'hash' + crypto.createHash('md5').update(JSON.stringify(col)).digest('hex').replace(/[0-9]/g, '');
-                items[i.desc].hashes.push(colHash);
-
-                if (col.type === 'enumeration') {
-
-                    let key = 'hash' + crypto.createHash('md5').update(JSON.stringify(col.enumerations)).digest('hex').replace(/[0-9]/g, '');
-
-                    if (!listEntries[key])
-                        listEntries[key] = col.enumerations;
-
-                    if (!combos[colHash]) {
-                        col.enumHash = key;
-                        combos[colHash] = col;
-                    }
-
-                } else {
-                    if (!texts[colHash])
-                        texts[colHash] = col;
-                }
-            });
+        const itemKey = i.desc + ' - ' + i.geom;
```

Comment:
with setting the key in the `items` object to be the `*desc* - *geom type*` we assure no duplicates but also was able to revert all the code changes i made recently because able to seperate items like `Dam - Area` and `Dam - Line` at this step and it will propogate